partials/messages.html

```diff
@@ -5,7 +5,7 @@ <h2 class="ml-8 mt-4">Messages</h2>
 
 <div class="responsive-container">
   <div class="center-block p-8">
-    <div class="pb-8">
+    <div class="message-examples pb-8">
```

Comment:
```suggestion
    <div class="all-messages pb-8">
```

these are not examples but all messages from all the channels